src/OrchardCore.Modules/OrchardCore.ArchiveLater/Drivers/ArchiveLaterPartDisplayDriver.cs

```diff
@@ -49,7 +49,7 @@ public override async Task<IDisplayResult> UpdateAsync(ArchiveLaterPart part, IU
 
             await updater.TryUpdateModelAsync(viewModel, Prefix);
 
-            if (viewModel.ScheduledArchiveLocalDateTime == null || httpContext.Request.Form["submit.Save"] == "submit.CancelArchiveLater")
+            if (viewModel.ScheduledArchiveLocalDateTime == null || httpContext.Request.Form["submit.Publish"] == "submit.CancelArchiveLater")
```

Comment:
If we do so, we need to change this too

https://github.com/OrchardCMS/OrchardCore/blob/242115849b741325b667050b40b78d947ea477f0/src/OrchardCore.Modules/OrchardCore.PublishLater/Drivers/PublishLaterPartDisplayDriver.cs#L57

Reply:
BTW this will break thing, because we should change the name in view

https://github.com/OrchardCMS/OrchardCore/blob/242115849b741325b667050b40b78d947ea477f0/src/OrchardCore.Modules/OrchardCore.ArchiveLater/Views/ArchiveLaterPart.Edit.cshtml#L18

I can fix it in both `OC.ArchiveLater` and `OC.PublishLater`

Reply:
No

Reply:
Could you please elaborate what's going on? How you change the submit button name in the driver without changing the view, I'm confused ;)

Reply:
Normally I changed it in the view too, I think you commented before I submitted my changes